.travis.yml

```diff
@@ -0,0 +1,31 @@
+sudo: required
+dist: trusty
+
+language: cpp
+
+compiler:  gcc
+
+before_install:
+- sudo apt-get update -qq
+- sudo apt-get install -qq libperl4-corelibs-perl libtool xorg-dev doxygen python-dev alien swig libgtest-dev libstdc++6 libxml2-dev gettext libxinerama-dev libxft-dev libxrandr-dev libxcursor-dev libgdbm-dev libc6 libc6-dev libmng-dev zlib1g-dev libcap-dev libpng12-0 libpng12-dev freeglut3-dev flex libx11-dev bison++ bisonc++ libjpeg-dev libjpeg8-dev python2.7 python2.7-dev python2.7-psycopg2 libogdi3.2-dev libgif-dev libxerces-c-dev libgeos-dev libgeos++-dev libfreetype6 libfreetype6-dev python-imaging libproj-dev python-setuptools libgif-dev libxerces-c-dev libcap-dev libpq-dev openssl libxml2-utils libxmu-dev
+
+
+before_script:
+  - cd earth_enterprise
+
+script:
+  - scons -j3 release=1 build >build.log
+  - cd src/NATIVE-REL-x86_64/bin/tests
+  - ./RunAllTests.pl
+
+cache:
```

Comment:
What is this cache about?
Seems like the notification is working in OpenGEE channel, however , it failed last time I saw the status.

can you attach a couple of screenshots of the results and Travis CI admin page or build output page?

Reply:
It looks like it's configured to run all the pull requests, including the ones that aren't properly configured for Travis CI. Those will always fail :)

But, the last time it ran from this branch, it got this error:
```
geRelativePathTester succeeded
Fusion Notice:	Closing file /tmp/khk5B0bO/big.txt.
Fusion Notice:	Copying created file: /tmp/khk5B0bO/big.txt
Fusion Warning:	Unable to write 1048576 bytes to /tmp/khk5B0bO/big.txt: No space left on device
ERROR: expected "kBuffSize * kTestFileSize" == "(size_t) statBuf.st_size
       actual  "kBuffSize * kTestFileSize" = 4294967296 , "(size_t) statBuf.st_size" = 3557388288
" at 
  common/tests/geFileUtils_test.cpp:121
ERROR: expected "num_read" == "kBuffSize
       actual  "num_read" = 0 , "kBuffSize" = 4096
" at 
  common/tests/geFileUtils_test.cpp:129
```

That might be something that should be fixed